main.groovy

```diff
@@ -55,7 +55,8 @@ stages {
                 string(credentialsId: 'WUM_PASSWORD', variable: 'wumPassword'),
                 string(credentialsId: 'DEPLOYMENT_DB_PASSWORD', variable: 'dbPassword'),
                 string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 's3accessKey'),
-                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 's3secretKey')])
+                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 's3secretKey'),
+                string(credentialsId: 'TESTGRID_EMAIL_PASSWORD', variable: 'TESTGRID_EMAIL_PASSWORD')])
```

Comment:
shall we rename the variable value to something like `testgridEmailPassword`? Also, is this specific to APIM?

Reply:
currently I think this is for APIM

Reply:
As this is email credentials, any product can use it send emails.